lib/cc/analyzer/container.rb

```diff
@@ -172,6 +172,7 @@ def kill_reader_threads
       def reap_running_container(message)
         Analyzer.logger.warn("killing container name=#{@name} message=#{message.inspect}")
         POSIX::Spawn::Child.new("docker", "kill", @name)
+        POSIX::Spawn::Child.new("docker", "wait", @name)
```

Comment:
You're right. I was originally going to argue against adding the complexity of a timeout here for a number of reasons, then I realized I can just pass the `:timeout` option to `Child.new` which is super easy. I'll do that.